packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts

```diff
@@ -1,12 +1,32 @@
-import { defineComponent } from 'vue'
-import ColorCSSVariablesUpdater from './ColorCSSVariablesUpdater.demo.vue'
+import { VaChip, VaDivider, VaButton } from '../../components'
+import { useColors } from '../../composables'
 
 export default {
-  title: 'ColorCSSVariablesUpdater',
-  component: ColorCSSVariablesUpdater,
+  title: 'services/ColorCSSVariablesUpdater',
 }
 
-export const Default = defineComponent({
-  components: { ColorCSSVariablesUpdater },
-  template: '<ColorCSSVariablesUpdater/>',
+export const Default = () => ({
+  components: { VaChip, VaDivider, VaButton },
+  methods: {
+    change (color) {
+      useColors().setColors({ danger: color })
+    },
+  },
+  template: `
+    <div class="mb-6">
+      <a class="title title--va-danger">
+        This is title danger color
+      </a>
+      <va-chip color="danger">This is vue component</va-chip>
+    </div>
+
+    <va-divider />
```

Comment:
The style guide for stories is kind of new, I'm not really into it right now, but va-diivder feels redundant here.